app/orm-adapters/node-orm2.js

```diff
@@ -7,7 +7,7 @@ import snakeCase from 'lodash/snakeCase';
 import upperFirst from 'lodash/upperFirst';
 
 const typeMap = {
-  text: String,
+  string: String,
```

Comment:
@davewasmer Just checked this again.  With my change above, at this point I'm getting JSON-API responses from the backend from a Sqlite DB! Without it, I get this error from `node-orm2`:

```
ERROR - AssertionError: "string" data type is not supported by the node-orm2 adapter. Supported types are: text, number, boolean, json, date
```

Those docs must be out of date, but the error above seems pretty clear.